src/Composition/ForceElements/Bushing.jl

```diff
@@ -123,29 +123,29 @@ mutable struct Bushing{F <: Modia3D.VarFloatType} <: Modia3D.AbstractForceElemen
                 springForceFunction[dir] = springForceLaw[dir]
             else
                 stiffness = Modia3D.convertAndStripUnit(F, u"N/m", springForceLaw[dir])
```

Comment:
I'm wondering, is `stiffness::F = Modia3D.convertAndStripUnit()` needed?

Reply:
Not needed of course, but nice to have.